Data/Datafunctions/data_availability.py

```diff
@@ -241,11 +241,23 @@ def general(c):
                 offered_type = c['summaries']['DataAvailability'][i]['offered_type']
             except Exception:
                 offered_type = ''
+            try:
+                mosaic = c['summaries']['DataAvailability'][i]['Mosaic']
+            except Exception:
+                mosaic = ''
+            try:
+                polarization = c['summaries']['DataAvailability'][i]['Polarization']
+            except Exception:
+                polarization = ''
+            try:
+                S3path = c['summaries']['DataAvailability'][i]['S3 Path']
+            except Exception:
+                S3path = ''
         
             if offered_type != '':
-                t.append([offered_type, status, access, spatial, temporal, available_from])
+                t.append([offered_type, status, access, spatial, temporal, available_from, mosaic, polarization, S3path,])
                 note += footnotes
-                headers = ["Timeliness","Archive Status", "Access Type", "Spatial Extent", "Temporal Extent", "Available in Ecosystem from"]
+                headers = ["Timeliness", "Archive Status", "Access Type", "Spatial Extent", "Temporal Extent", "Available in Ecosystem from", "Mosaic", "Polarization", "S3 Path", ]
```

Comment:
It seems to work as expected.

However @Kampen1988 @azlinszkysinergise these cells "mosaic, polarization" though added, isn't reflected in the table.

Reply:
@Pratichhya , who can I follow up with on the missing table columns and cells? Kaner?

Reply:
sure 
cc @KanerLev

Reply:
Hi @KanerLev ,
would you be available to provide some guidance on the necessary changes to `data_availability.py` if we want the currently missing columns and cells to be rendered?

Reply:
@Pratichhya, is there anyone else who could help me out with this?

Reply:
Hi @Kampen1988, Pratichhya is not available. 
Yes, I'll be able to help as much as I can. I couldn't write earlier.
Sent you a meeting request, feel free to adjust.

Reply:
Solved